tools/newrelic-infra-public-keys/newrelic-infra-public-keys

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+KEYRING_FILE_NAME=newrelic-infra-keyring.gpg
+TRUSTED_GPG_D_KEYRING=/etc/apt/trusted.gpg.d/${KEYRING_FILE_NAME}
+USR_SHARE_KEYRING=/usr/share/keyrings/${KEYRING_FILE_NAME}
+KEYS_LOCATION=/opt/newrelic-infra-public-keys
```

Comment:
`/opt` has a special meaning, we could alternative use something in the usual dir `/var/db/newrelic...`